### PR TITLE
add missing functions

### PR DESCRIPTION
**Pull request by cmeissl**
This adds all missing functions exposed by the ffi library

### CODE REVIEW

**Comment by YaLTeR** [comment]
So what is actually left to do in this PR, what kind of help is needed? (I am not familiar with libd-i or these bindings.) Also, from a glance, I guess this supersedes #3?

**Comment by cmeissl** [comment]
> So what is actually left to do in this PR, what kind of help is needed? (I am not familiar with libd-i or these bindings.) Also, from a glance, I guess this supersedes #3?

I am unhappy with the current state of the proc-macro. Yes, this supersedes #3

File: libdisplay-info/src/edid.rs [hunk 1]
```diff
@@ -10,28 +16,786 @@ pub struct Edid<'info> {
...
+    ///
+    /// Returns `None` if this isn't a digital display.
+    pub fn video_input_digital(&self) -> Option<VideoInputDigital> {
+        VideoInputDigital::from_ptr(unsafe {
+            ffi::edid::di_edid_get_video_input_digital(self.edid)
+        })
+    }
+
+    /// Get the screen size.
+    pub fn screen_size(&self) -> ScreenSize {
```
Comment (i509VCB) [inline libdisplay-info/src/edid.rs:51]: In mm?

Comment (cmeissl) [reply libdisplay-info/src/edid.rs:51]: The returned `ScreenSize` contains the size in `cm`, like the underlying ffi type:

```
pub struct di_edid_screen_size {
    pub width_cm: ::std::os::raw::c_int,
    pub height_cm: ::std::os::raw::c_int,
    pub landscape_aspect_ratio: f32,
    pub portait_aspect_ratio: f32,
}
```

**Pull request merged**